ui/app/templates/jobs/run/templates/index.hbs

```diff
@@ -1,10 +1,26 @@
 <section class="section">
-    <h1 class="title with-flex">Hey buddy!</h1>
-    <span>Here are some preset template we create for you to get started. You can also create custom templates for you or your team to use in the future.</span>
-    <main data-test-template-list>PLACEHOLDER FOR RADIO CARD GROUP</main>
-    {{!-- Add Conditional Logic when no templates available --}}
-    <div>
-        <button data-test-apply>PLACEHOLDER USE HDS</button>
-        <button data-test-cancel>PLACEHOLDER USE HDS</button>
-    </div>
+    <header class="run-job-header">
+        <h1 class="title is-3">Choose a template</h1>
+        <p>A plan will be requested before the job is submitted.  You can pick a template or write your own job spec.</p>
+    </header>
+    <main class="radio-group" data-test-template-list>
+        <Hds::Form::RadioCard::Group @alignment="left" as |G|>
+            <G.Legend>Select a Template</G.Legend>
```

Comment:
Let's move "Select a template" into a conditional for if templates exist

<img width="1036" alt="image" src="https://user-images.githubusercontent.com/713991/211349101-447816d0-4cf8-42ae-9c05-f6ae34ea677c.png">

Reply:
Is this what you're looking for

![image](https://user-images.githubusercontent.com/41024828/211376423-6c694d20-4436-436e-b332-605a60e014d4.png)

Reply:
That looks great!